blocks/floatingagent/floatingagent.css

```diff
@@ -0,0 +1,81 @@
+.floatingagent.block {
+    display: none;
+    align-items: center;
+    justify-content: space-between;
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 100px;
+    background-color: var(--white);
+    padding: 10px;
+    border-top: 1px solid #ccc;
+    z-index: 999;
+}
+
+.floatingagent.block > .floating-agent-col {
+    margin-left: 100px;
+    display: flex;
+    margin-right: 10px;
+    flex: none;
+}
+
+.floatingagent.block > .agentinfo > p {
+    margin-block-end: 0;
+    font-family: var(--font-family-primary);
+    font-size: var(--body-font-size-xxs);
+    font-style: normal;
+    font-weight: var(--font-weight-normal);
+    letter-spacing: normal;
+    color: #2a2223;
+}
+
+.floatingagent.block > .agentinfo > h2 {
+    font-size: var(--heading-font-size-m);
+    line-height: 130%;
+    letter-spacing: .18px;
+    vertical-align: top;
+    color: #2a2223;
+    width: max-content;
+}
+
+.floatingagent.block > .floating-agent-col > picture {
+    height:40px;
+    width:30px;
+    margin-right: 40px;
+}
+
+.floatingagent.block > .floating-agent-col > picture > img{
+    display: inline;
+}
+
+.floatingagent.block > .agentinfo {
+    flex-grow: 1;
+    margin-top: 10px;
+}
+
+.floatingagent.block > .contactagent {
+    align-self: center;
+    background: var(--primary-color);
+    color: var(--tertiary-color);
+    border: 0;
+    flex: none;
+    padding: 10px;
+    margin-right: 5%;
+}
+
+@media (max-width: 600px) {
```

Comment:
can you flip this? always start with mobile view as the default, then it's `min-width` on up. I realize that will probably have pretty significant changes to the styling.

Reply:
Corrected.